examples/serverless-example/serverless.yml

```diff
@@ -28,7 +28,17 @@ provider:
   runtime: nodejs8.10
   stage: development
   region: eu-west-1
-  profile: sandbox
+  profile: sandbox # TODO You need to either remove this or use your profile
+  environment:
+    TABLE_NAME: Book
+
+functions:
+  listBackups:
+    handler: src/listBackups.handler
+    events:
+      - http:
+          method: GET
+          path: /listBackups
```

Comment:
Cool!

Should probably be documented in the README, it's not obvious there is an available endpoint ;)